.github/workflows/snapshot.yaml

```
@@ -100,6 +100,7 @@ jobs:
     name: Build Docker image
     runs-on: ubuntu-latest
     needs: [Lint, Format, Build, Test]
+    needs: [Lint, Format, Build, Test]
```

actionlint (1.7.12)

[error] key "needs" is duplicated in "Push-Image" job. previously defined at line:102,col:5

(syntax-check)